courses/free-courses-en.md

```diff
@@ -307,6 +307,7 @@
 * [Bash Scripting Full Course 3 Hours](https://www.youtube.com/watch?v=e7BufAVwDiM) - Linuxhint
 * [Bash Scripting Tutorial](https://ryanstutorials.net/bash-scripting-tutorial/) - Ryans Tutorial
 * [Bento Shell Track](https://bento.io/topic/shell) - Jon Chan (Bento)
+* [Bourne Shell Programming / Scripting Tutorial](https://www.shellscript.sh) - The Shell Scripting Tutorial
```

Comment:
Belongs in books, I think.
Don't invent titles! Use the title given by the resource.

Reply:
Got it, I'll move it to the books section and use the exact title provided by the resource. Thanks for the clarification!

Reply:
you've left it in courses